### DISCUSSION TITLE
Error checking image in documents written in a language other than English

### DISCUSSION THREAD

**Discussion by TheAwiteb** [community/community#29656] [GitHub Education]
Hello, I got my university card today and I wanted to send a request for the Github educational bag because it will help me in my development career, GitHub recognized my university email and showed the name of the university, and when I presented the bag it asked me for a picture with my card and I took a very clear picture,  But he refused and that was the reason

![IMG_20220816_124506](https://user-images.githubusercontent.com/59842932/184850752-78d061d1-5a32-4ccf-9f4f-763d72f5e91f.jpg)

**Comment by mazkdevf** [comment]
Hello TheAwaiteb,

Did you provide an clear image that clearly shows the requirements?

**Reply by TheAwiteb** [reply]
Yes, all requirements are present and everything is clear

**Reply by mazkdevf** [reply]
Okay i think you should contact https://support.github.com/

**Reply by LiliuZhao09** [reply]
I also encountered the same problem, and I provided my official website certificate of student status to Github，But he refused，For the same reason you did

**Comment by LiliuZhao09** [comment]
I have applied to GIT four times and have been rejected each time for the
same reason🙁

Awiteb ***@***.***> 于2022年8月18日周四 15:03写道：

**Comment by delosemre** [comment]
Have you added your student email address to Github?

**Reply by TheAwiteb** [reply]
> Have you added your student email address to Github?

Yes of course, it's end with (stu.kau.edu.sa)

**Reply by TheAwiteb** [reply]
btw, I tried to contact GitHub but didn't get a response, I'll try contacting them on Twitter. Someone told me they were responsive there.

**Reply by delosemre** [reply]
What is the document you uploaded? According to the image above, it says the date, school name and name are not visible. submitting your transcript should be sufficient.

**Reply by TheAwiteb** [reply]
It's school id. I think the reason for that is because it is written in Arabic

**Reply by TheAwiteb** [reply]
The date is in English, which is 2029, but the rest of the information is in Arabic

**Reply by TheAwiteb** [reply]
And I think @LiliuZhao09  the same problem is written in a language other than English

**Reply by delosemre** [reply]
Sorry mate. All you have to do is create a support request.
https://support.github.com/ You can open a ticket here.
There seems to be no other solution.

**Comment by LiliuZhao09** [comment]
-

   Yes, the certificate I submitted was indeed in Chinese


Awiteb ***@***.***> 于2022年8月23日周二 19:09写道：
